commands/initial/templates/webpack/webpack.dev.js

```diff
@@ -29,7 +29,6 @@ module.exports = webpackCommon('dev', {
     devtool: 'cheap-module-eval-source-map',
     entry: {
         app: [ examplePath('example.main') ],
-        scripts: [],
```

Comment:
Webpack reports the following for me when trying to run it

WebpackOptionsValidationError: Invalid configuration object. Webpack has been initialised using a configuration object that does not match the API schema.
 - configuration.entry should be one of these:
   object { <key>: non-empty string | [non-empty string] } | non-empty string | [non-empty string] | function
   The entry point(s) of the compilation.
   Details:
    * configuration.entry['scripts'] should be a string.
    * configuration.entry['scripts'] should not be empty.
    * configuration.entry['scripts'] should be one of these:
      non-empty string | [non-empty string]
    * configuration.entry should be a string.
    * configuration.entry should be an array:

I'm not sure what to do about this one, but it looks like a misconfiguration?